src/directives/formioWizard.js

```diff
@@ -445,6 +445,17 @@ module.exports = function() {
 
         // Move onto the previous page.
         $scope.prev = function() {
+          var errors = $scope.checkErrors();
+          if (errors) {
+            $scope.pageHasErrors[$scope.currentPage] = true;
+            if (!$scope.formioOptions.wizardFreeNavigation) {
```

Comment:
Hi Travis,

It was already there in the wizard directive.  Allows user to skip around between pages in a non-linear fashion.